### PR TITLE
Fix build

### PR DESCRIPTION
**Pull request by dkfellows**
Fixes to the build:
* Use Python 3.8
* Use correct egg-links
* Use the correct set of filters for the contents
* Freshen the content of the index page (enhancement!)

See https://readthedocs.org/projects/spinnaker8manchester/builds/13088101/ for evidence of correctness.

### CODE REVIEW

**Review by rowleya — APPROVED** [review]
Looks OK - bearing in mind there is a list of files to keep up to date! :)

File: doc/source/index.rst [hunk 1]
```diff
@@ -1,16 +1,26 @@
 SpiNNaker Manchester (PyNN 0.9)
 =============================
-These pages document the main Manchester python code for the PyNN 0.9 branch of the SpiNNaker_ Project which can be found on github_
-
-Alternative way to run SpiNNaker_ are using the SpiNNakerGraphFrontEnd_.
-
+These pages document sPyNNaker 6, an implementation of PyNN_ 0.9 for the
```
Comment (Christian-B) [inline doc/source/index.rst:3]: sPyNNaker 6 ????

Comment (Christian-B) [reply doc/source/index.rst:3]: Do we actually still need to say it is PyNN 0.9 ?

**Review by Christian-B — DISMISSED** [review]
minor changes.

**Comment by Christian-B** [comment]
It would be good to get the 8 out of https://spinnaker8manchester.readthedocs.io

**Comment by dkfellows** [comment]
> It would be good to get the 8 out of https://spinnaker8manchester.readthedocs.io

I think that's set through the RTD web API.

File: doc/source/index.rst (hunk 1 above)
Comment (dkfellows) [reply doc/source/index.rst:3]: I don't really care too much either way.

**Pull request merged**